libs/platform/src/lib/components/menu/menu.component.ts

```diff
@@ -133,29 +135,21 @@ export class MenuComponent implements OnInit, AfterViewInit, AfterContentInit, O
     }
 
     onKeydown($event: KeyboardEvent): void {
```

Comment:
https://github.com/SAP/fundamental-ngx/pull/5673/files#r655965061

Reply:
made it uniform with other components. using `event`, removed `$event`.